services/preview/videoanalyzer/mgmt/2021-05-01-preview/videoanalyzer/version.go

```diff
@@ -1,3 +1,5 @@
+// Deprecated: We’re retiring the Azure Video Analyzer preview service, you're advised to transition your applications off of Video Analyzer by 01 December 2022. This SDK is no longer maintained and won’t work after the service is retired
```

Comment:
```suggestion
// Deprecated: We’re retiring the Azure Video Analyzer preview service, you're advised to transition your applications off of Video Analyzer by 01 December 2022. This SDK is no longer maintained and won’t work after the service is retired.
```

One package needs only one deprecation message. I think you could reserve this one.